code/game/gamemodes/cult/cult_items.dm

```diff
@@ -682,3 +682,107 @@
 	desc = "It's an orb of crystalized blood. Can be used to transfer blood between cultists."
 	var/blood = 50
 
+/obj/item/portal_amulet
+	name = "reality sunderer"
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "amulet"
+	desc = "Some amulet made out of metal, bluespace crystals, and blood. Allows cultists to open portals over teleport runes, destroying the rune in the process."
+	w_class = WEIGHT_CLASS_SMALL
+
+
+/obj/item/portal_amulet/afterattack(atom/O, mob/user, proximity)
+	. = ..()
+	if(!iscultist(user) && iscarbon(user))
```

Comment:
```suggestion
	if(!iscultist(user))
		if(!carbon(user))
			return
```
Otherwise, the rare case still stays open where a non-carbon somehow manages to use this amulet and it succeeds since they ain't carbon nor a cultist.
Could also make it so that they don't get blinded or something but still have the teleport happen